index.bs

```diff
@@ -2489,6 +2490,9 @@ The [=remote end steps=] with |command parameters| are:
   1. Let |context id| be the value of the <code>context</code> field of
      |command parameters|.
 
+  1. Let |prompt unload| be the value of the <code>promptUnload</code> field of
+     |command parameters|.
```

Comment:
Since the parameter is optional I think we need to check if |command parameters| contains "promptUnload" first?

Reply:
My mistake, since there is a default value this is probably not needed.